std_msgs/Header header

uint32 num_vectors

geometry_msgs/Vector3[] tails
geometry_msgs/Vector3[] tips
